spec/support/schemas/claims_api/v2/veterans/claims/claims.json

```diff
@@ -69,6 +69,50 @@
         "nullable": true,
         "description": "If true, indicates a decision has been requested and/or a Waiver 5103 has been submitted",
         "example": "false"
+      },
+      "trackedItems": {
+        "type": "array",
+        "description": "",
+        "items": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["closedDate", "description", "dvlpmtTc", "suspensionDate", "requestedDate", "trackedItemId"],
+          "properties": {
+            "closedDate": {
+              "type": "string",
+              "format": "date",
+              "description": "Date the tracked item was closed",
+              "example": "2022-01-01T00:00:00-04:00"
+            },
+            "description": {
+              "type": "string",
+              "description": "Description of the tracked item",
+              "example": "Buddy mentioned - No complete address"
+            },
+            "dvlpmtTc": {
+              "type": "string",
+              "description": "",
```

Comment:
I would like a value for this field, but we don't know what to put there (see [thread](https://oddball.slack.com/archives/C019XKGS1CN/p1660683310668549))

Reply:
We actually don't want any of these `trackedItems` properties to show up in the claims index response.
See previous comment about adding an `exclude` to the blueprint so this new property doesn't show up in claims index.

Reply:
Now that you've added the `exclude`, you'll need to re-run rswag so that this schema is updated to no longer include `trackedItems`.  In the end, this `claims.json` file shouldn't have any changes in this PR.